codes.json

```diff
@@ -743,21 +743,22 @@
   ,
   {
     "title" : "Osiris",
-    "subtitle" : "TBD/",
-    "description" : "TBD/",
-    "application": [],
-    "compute": [],
-    "method": [],
+    "subtitle" : "an object-oriented parallel 3D PIC code for modeling laser and particle beam-plasma interactions",
+    "description" : "An object-oriented, high performance fully explicit EM-PIC code",
+    "application": ["plasma based accelerators", "radiation sources"],
+    "compute": ["laptop/desktop", "linux clusters", "HPC centers"],
+    "method": ["PIC"],
     "phenomena": [],
     "homepage" : "http://picksc.idre.ucla.edu/software/software-production-codes/osiris",
```

Comment:
@tsung1029 I just realized this homepage link is broken, do you potentially have a  new one? :)

Reply:
Thanks, sounds great!

Reply:
The web-site is picksc.physics.ucla.edu, but it is not up yet.  I will update it as soon as the website become available.